service/controller/clusterapi/drainer.go

```diff
@@ -16,7 +16,8 @@ import (
 	"sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset"
 
 	awsclient "github.com/giantswarm/aws-operator/client/aws"
-	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v27"
+	v27 "github.com/giantswarm/aws-operator/service/controller/clusterapi/v27"
+	v28 "github.com/giantswarm/aws-operator/service/controller/clusterapi/v28"
```

Comment:
Please don't. Further, we can drop v27 when we have v28.

Reply:
Sorry. I did not do on purpose. VS autocompletion